bin/init.bash

```diff
@@ -196,6 +189,7 @@ set_elasticsearch_config() {
     esac
 
     replace_set_me "$1" 'elasticsearch.useRegionEndpoint' "$use_regionendpoint"
+    replace_set_me "$1" 'elasticsearch.snapshotRepository' "s3_${CK8S_CLOUD_PROVIDER}_7.x"
```

Comment:
Why do we need the cloud provider and version in this? Isn't this only the name of the repo?
```suggestion
    replace_set_me "$1" 'elasticsearch.snapshotRepository' "s3_repository"
```